File: Projetos/Políticos/C++/sources/DeputadoEstadual.cpp
// DeputadoEstadual.cpp
#include <stdio.h>
#include <stdlib.h>
#include <iostream>
#include "../includes/DeputadoEstadual.h"
using namespace std;

DeputadoEstadual::DeputadoEstadual(string nome,string partido, string estado) {
    setNome(nome);
    setSalario(10000);
    setPartido(partido);
    setEstado(estado);
    setFuncao("propor as leis estaduais de interesse da população ");
}
DeputadoEstadual::~DeputadoEstadual() {
    setNome("");
    setPartido("");
    setEstado("");
}
void DeputadoEstadual::apresentacao(){
    Politico::apresentacao();
    cout << "sou deputado estadual" << endl;
    cout << "Minha função é " << getFuncao();
    cout << getEstado() << endl;
    cout << "============================" << endl;
}

File: 07/Python3/DeputadoEstadual.py
#!/usr/bin/python3
# coding: utf-8
"""Classe DeputadoEstadual """

from Politico import Politico


class DeputadoEstadual(Politico):
    """ Classe Deputado Estadual """

    def __init__(self, nome, partido, estado):
        """ Construtor da classe DeputadoEstadual """
        Politico.__init__(self)
        self.set_nome(nome)
        self.set_partido(partido)
        self.set_estado(estado)
        self.set_funcao("propor as leis estaduais de interesse da população ")

    def apresentacao(self):
        super(DeputadoEstadual, self).apresentacao()
        print ('sou deputado estadual')
        print ('Minha função é ' + self.get_funcao())
        print ('Fui eleito por ' + self.get_estado())
        print ('============================')

